CMakeLists.txt

```diff
@@ -1,6 +1,27 @@
-project( common_clang )
 cmake_minimum_required(VERSION 3.4.3)
 
+if(NOT DEFINED BASE_LLVM_VERSION)
+  set (BASE_LLVM_VERSION 11.0.0)
+endif(NOT DEFINED BASE_LLVM_VERSION)
+set(OPENCL_CLANG_VERSION ${BASE_LLVM_VERSION}.0)
+
+if (NOT DEFINED OPENCL_CLANG_BUILD_EXTERNAL)
```

Comment:
please be consistent whether there is a space after if

Reply:
done, thanks.